### ISSUE TITLE
No longer works since k8s remove moved from Google to Community

### ISSUE THREAD

**Issue by doritoes** [torgeirl/kubernetes-playbooks#27]
This was a super-helpful playbook to set up Kubernetes. No longer works since early 2024 changes to move k8s repo from Google to Community.

I'm trying to apply the tips from https://medium.com/@martin.hodges/installing-kubernetes-from-the-new-k8s-io-repository-using-ansible-8e7319fa97fd to update the yaml files on my own.

**Comment by torgeirl** [comment]
Hi @doritoes,

> No longer works since early 2024 changes to move k8s repo from Google to Community.

I changed the playbooks to use the community repos in December (f76d8a116f8bab78f6a8ce99eff539b7854fbdd8). I just ran them on a few Ubuntu 22.04 LTS VMs and got a Kubernetes v1.29.4 cluster without any issues. This should be the result using either `main` (just renamed from `master`), `v1.28` and `v1.29` branches. Perhaps you haven't pull the changes from GitHub?

Edit: [your fork](https://github.com/doritoes/kubernetes-playbooks) seems to be a little behind («This branch is 5 commits behind torgeirl/kubernetes-playbooks:main.») :wink:
